app/CocoaPods/Base.lproj/MainMenu.xib

```diff
@@ -134,6 +134,13 @@
                                     </items>
                                 </menu>
                             </menuItem>
+                            <menuItem isSeparatorItem="YES" id="wV5-hB-goc"/>
+                            <menuItem title="Remove CocoaPods From A Project..." id="fXB-fZ-GrP">
```

Comment:
can you make this: "Remove CocoaPods From a Project…" ( lowercase a, and single char …)